common/src/main/java/whocraft/tardis_refined/common/blockentity/device/ManipulatorCraftingRecipe.java

```diff
@@ -0,0 +1,46 @@
+package whocraft.tardis_refined.common.blockentity.device;
+
+import net.minecraft.world.item.Item;
+import net.minecraft.world.level.block.Block;
+
+import java.util.Comparator;
+import java.util.List;
+
+public class ManipulatorCraftingRecipe {
```

Comment:
This class could do with some documentation

Reply:
Slapped a little documentation in.